config/locales/en.yml

```diff
@@ -39,6 +39,15 @@ en:
     posts: Posts
     remote_follow: Remote follow
     unfollow: Unfollow
+  activitypub:
```

Comment:
This is an area I don't know enough about. I did this just to be sure, but I'll run through the AP documentation again to check if there's a need for internationalization.